docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1963 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #93.                is_cands = _find_candidates(T, m, M_T, Σ_T, min_dist, init_cands = init_cands, right = True, finite = finite)
</pre><p>FYI, in Python function calls, the convention is to remove spaces before and after the <code>=</code> sign </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
Thanks for this tip! I will  fix it.